client/src/components/settings/portfolio.tsx

```diff
@@ -286,6 +287,7 @@ class PortfolioSettings extends Component<PortfolioProps, PortfolioState> {
               value={image}
             />
             {imageMessage ? <HelpBlock>{imageMessage}</HelpBlock> : null}
+            {image && <img src={image} className='preview-image' alt='' />}
```

Comment:
@Sboonny 
Here I am rendering it because if we don't do this then once we clear the image url, it will still display an empty image box which is not what we want

Reply:
Sorry for the delay about this, I am waiting for some feedback about how the image can be actually shown.

while we are waiting for the feedback, can you add alt message as following 

```ts
alt=`preview image for ${title}`
```

Reply:
> Sorry for the delay about this, I am waiting for some feedback about how the image can be actually shown.
> 
> while we are waiting for the feedback, can you add alt message as following
> 
> ```ts
> alt=`A visual preview image for ${title}`
> ```

Okay ,will add that and push the change

Reply:
no need, it isn't a necessary yet, because it's currently for visual users

Reply:
What else is required to do?
Also all checks are passing now